src/platform/packages/shared/kbn-esql-ast/src/definitions/utils/autocomplete/helpers.ts

```diff
@@ -345,6 +348,7 @@ export async function suggestForExpression({
   location,
   preferredExpressionType,
   context,
+  advanceCursorInitially = true,
```

Comment:
What is this `advanceCursorInitially ` property? Why is not `advanceCursor`?

Reply:
It only applies to the first field that is inserted. Maybe `advanceCursorAfterInitialField`?

Reply:
aaah I see, yes `advanceCursorAfterInitialField ` much cleaner 🙌

Reply:
Done in https://github.com/elastic/kibana/pull/223176/commits/6b50c827f5c7eedf8a531b18acbdbf7d243aecf2